services/shared/src/main/java/com/wedriveu/services/shared/rabbitmq/VerticleConsumer.java

```diff
@@ -32,11 +33,13 @@ public VerticleConsumer(String name) {
     public void start() throws Exception {
         this.queueName = Constants.SERVICE_QUEUE_BASE_NAME + "." + name;
```

Comment:
Can't we make it more flexible? So, not hard-coding the `SERVICE_QUEUE_BASE_NAME` before the actual name but let the subclass decide the complete name?
And alsom why there's an `@author nicolalasagni`? I didn't create/edit this file.